lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -1,45 +1,44 @@
 import { FullScreenSpinner, LabeledButton } from '@hypothesis/frontend-shared';
 import { useMemo, useState } from 'preact/hooks';
 
+import type { Book, File, Chapter } from '../api-types';
 import { useConfig } from '../config';
 import { PickerCanceledError } from '../errors';
+import type { Content } from '../utils/content-item';
 import { GooglePickerClient } from '../utils/google-picker-client';
 import { OneDrivePickerClient } from '../utils/onedrive-picker-client';
+import type { ErrorInfo } from './FilePickerApp';
```

Comment:
This import should probably be grouped with the next set of imports (it is alphabetically after `BookPicker`).